### ISSUE TITLE
Accessibility: Audit Sign Up and Sign In flow

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-Android#7281]
Audit the Sign-Up and Sign-In flow to make sure that all the functionality is accessible with TalkBack and Switch Access.

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.
